src/test/test-seccomp.c

```diff
@@ -91,7 +91,14 @@ static void test_syscall_filter_set_find(void) {
         assert_se(syscall_filter_set_find("@raw-io") == syscall_filter_sets + SYSCALL_FILTER_SET_RAW_IO);
 }
 
+#if HAS_FEATURE_MEMORY_SANITIZER
+/* As we don't have the intrumented version of the seccomp libarry in CIs,
+ * disable the memory sanitizer for this function to avoid false positives
+ */
+__attribute__((no_sanitize("memory"))) static void test_filter_sets(void) {
+#else
 static void test_filter_sets(void) {
+#endif
```

Comment:
The `#else` branch is not needed. In the `#if` branch, remove the part that starts with " static ".

Reply:
That commit should probably be removed altogether. `-Dseccomp=false` will make the issue go away.